src/plugin.js

```diff
@@ -296,31 +304,22 @@ class VR extends Plugin {
     }
 
     this.controls3d.update();
-    this.manager.render(this.scene, this.camera);
+    this.effect.render(this.scene, this.camera);
 
-    if (!this.vrDisplay) {
-      this.animationFrameId_ = this.requestAnimationFrame(this.animate_);
-      this.camera.getWorldDirection(this.cameraVector);
-      return;
-    }
-
-    this.animationFrameId_ = this.vrDisplay.requestAnimationFrame(this.animate_);
+    this.animationFrameId_ = this.requestAnimationFrame(this.animate_);
 
-    if (!window.navigator.getGamepads) {
-      return;
-    }
-    // Grab all gamepads
-    const gamepads = window.navigator.getGamepads();
+    if (window.navigator.getGamepads) {
+      // Grab all gamepads
+      const gamepads = window.navigator.getGamepads();
```

Comment:
This was just a refactor, as it didn't really make sense the way it used to work